### ISSUE TITLE
Token Management functions are over-engineering

### ISSUE THREAD

**Issue by Denisthemalice** [ietf-wg-gnap/gnap-core-protocol#293]
Token Management (section 6) is dealing with:
-	Rotating the Access Token, and
-	Revoking the Access Token.

If the access token has expired, it is simpler for the client instance to ask for a new access token rather than for a “rotation” (??).

If the access token has been revoked, rather than “indicating to the AS that the AS should invalidate the access token for all purposes”, the client instance should delete the access token from its context and from the context of the RS.

**Comment by jricher** [comment]
These are optional functions for advanced clients and therefore do not place burden on clients that do not want to do them.

**Comment by Denisthemalice** [comment]
Yes, these are optional functions, but this does not prevent to mention these alternatives in the document for "less advanced" clients.

**Comment by jricher** [comment]
The alternatives are simply not doing the optional functions, and this is already mentioned.
